### DISCUSSION TITLE
Do I need to add a css reset config?

### DISCUSSION THREAD

**Discussion by abdelrazzaq-dev** [adobe/react-spectrum#3332] [Q&A]
The red line is a margin added by an h3 element, do I need to add a CSS reset config along with the library to remove these spaces?
![image](https://user-images.githubusercontent.com/23277063/180429199-00798350-7445-4848-a419-e3e1cc00f1c2.png)

**Comment by snowystinger** [comment]
Sorry, I don't have much idea of what you've done. Could you reproduce it in a codesandbox? here's one to get started https://codesandbox.io/s/practical-fog-thg8me
